data-serving/data-service/src/common/db-client.ts

```diff
@@ -0,0 +1,24 @@
+import { MongoClient, Db } from 'mongodb';
+
+/** A thin wrapper around [[`MongoClient`]]. */
+export default class DbClient {
+    client: MongoClient;
+
+    constructor(connectionString: string) {
+        this.client = new MongoClient(connectionString, {
+            useUnifiedTopology: true,
+        });
+    }
+
+    public async connect() {
```

Comment:
Uh oh. Time to decide if we'll use explicit `public` (since it's default in TS). I vote implicit!

Reply:
gtk, agreed we should omit if it's default